.github/workflows/codeql.yml

```diff
@@ -0,0 +1,48 @@
+name: CodeQL
+
+on:
+  pull_request:
+  workflow_dispatch:
+    branches: [ "master" ]
```

Comment:
Not sure about `master` only. Sometimes it is useful to check another branch, stable like 2.10 or a dev one.

Reply:
I think we should start off with master. I all submit separate PR for release branches.